src/main/java/co/rsk/bitcoinj/script/FlyoverRedeemScriptParser.java

```diff
@@ -0,0 +1,70 @@
+package co.rsk.bitcoinj.script;
+
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.bitcoinj.core.VerificationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class FlyoverRedeemScriptParser {
+    private static final Logger logger = LoggerFactory.getLogger(FlyoverRedeemScriptParser.class);
+    protected final byte[] derivationHash;
+
+    public FlyoverRedeemScriptParser(
+        Script redeemScript,
+        Sha256Hash derivationHash
+    ) {
+        this.derivationHash = derivationHash.getBytes();
```

Comment:
Are we storing the derivationHash as bytes instead of Sha256Hash for any particular reason?

Reply:
nope, it just was that way before too